README.md

```diff
@@ -0,0 +1,63 @@
+
+# Software Factory Blueprint
+PELUX Software Factory Blueprint documentation
+
+Software Factory Blueprint is maintained at https://github.com/Pelagicore/software-factory-blueprint
+
+Maintainer: Joakim Gross <joakim.gross@pelagicore.com>
+
+
+## Dependencies
+* Sphinx
+* sphinxcontrib-seqdiag
+* sphinxcontrib-blockdiag
+* sphinxcontrib-actdiag
+* sphinxcontrib-manpage
+* sphinx\_rtd\_theme
+* texlive-latex-base (when building PDF)
+* texlive-latex-extra (when building PDF)
+* texlive-latex-recommended (when building PDF)
+
+###  Install build dependencies on Debian
+
+```
+sudo apt-get install python-pip
+pip install sphinxcontrib-seqdiag sphinxcontrib-blockdiag \
+              sphinxcontrib-actdiag sphinxcontrib-actdiag \
```

Comment:
Oh yeah I manage to forget to pull your changes in that repo before using it as a base for this. Will fix.